ecosystem/sep-0024.md

```diff
@@ -331,6 +333,13 @@ target.postMessage({
 
```

Comment:
Immediately above here is a good section that details the callback parameter and what data is posted, etc. I think we need a similar section for the new parameter. You could mostly copy paste it and note how the `on_change_callback` is posted to on every change.